### PR TITLE
Issue 13930: Add mpJwtPropagation-1.2 auto feature

### LINKED ISSUE #13930: Create new com.ibm.websphere.appserver.mpJwtPropagation-1.2 auto feature
The existing `com.ibm.websphere.appserver.mpJwtPropagation-1.0` feature doesn't appear to be sufficient for usage with `mpJwt-1.2`, so a new auto feature should be created.

### PR DESCRIPTION
**Pull request by ayoho**
Resolves #13930

### CODE REVIEW

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

**Comment by ayoho** [comment]
#run-libby-bot
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

File: dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpJwtPropagation-1.2.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpJwtPropagation-1.2
+visibility=private
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxrsClient-2.0)(osgi.identity=com.ibm.websphere.appserver.jaxrsClient-2.1)))", \
```
Comment (andymc12) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpJwtPropagation-1.2.feature:4]: do we need to support jaxrsClient-2.0?  I thought MP features all prereq EE8 (jaxrs/jaxrsClient-2.1).

**Comment by ayoho** [comment]
#run-libby-bot
#build
#spawn.fullfat.buckets=com.ibm.ws.security.jwt_fat.builder,com.ibm.ws.security.jwt_fat.consumer,com.ibm.ws.security.jwtsso_fat,com.ibm.ws.security.mp.jwt.1.1_fat,com.ibm.ws.security.mp.jwt.1.2_fat

**Review by andymc12 — APPROVED** [review]
LGTM - thanks @ayoho!

**Pull request merged**